README.md

```diff
@@ -541,6 +541,18 @@ config :wallaby,
   ]
 ```
 
+### Custom Chrome binary
+
+By default chromedriver will find chrome for you but perhaps you want to test against a different version
```

Comment:
Can we change the wording of this to just "but if you want to test against a different version..."